internal/secret/restricted/store_test.go

```diff
@@ -45,28 +46,21 @@ func TestBlockingTags(t *testing.T) {
 
 			secretID, err := store.Store(orgID, tc.request)
 			if err != nil {
-				t.Errorf("error during storing readonly secret: %s", err.Error())
-				t.FailNow()
+				t.Fatalf("error during storing readonly secret: %s", err.Error())
 			}
 
 			err = store.Delete(orgID, secretID)
 			if err == nil {
-				t.Error("readonly secret deleted..")
-				t.FailNow()
```

Comment:
Initially, this test was found to be faulty because the subsequent code was unreachable because of `t.FailNow()` in line 55, and during the modification process, it was found that the test should be refactored directly